src/demo-app/tabs/tabs-demo.ts

```diff
@@ -133,3 +133,14 @@ export class RainyTabContent {}
   template: 'This is the routed body of the foggy tab.',
 })
 export class FoggyTabContent {}
+
+@Component({
+  moduleId: module.id,
+  selector: 'counter',
+  template: `<span>{{count}}</span>`
```

Comment:
`count` variable is not defined

Reply:
@amcdnl this is a presubmit error in Google, so if you can fix that it should unblock moving forward

Reply:
Fixed.